deployments/deploy-ssh.sh

```diff
@@ -101,7 +101,7 @@ if [ -z "${TIMEOUT}" ]; then
 fi
 
 if [ -z "$DRIVER_START_TIMEOUT" ]; then
-    DRIVER_START_TIMEOUT="5m0s"
+    DRIVER_START_TIMEOUT="12m0s"
```

Comment:
Lets keep this to 10 mins for now!! in case Portworx comes up after 10 mins and before 10 mins test will resume and will hit issues in applications. Its better to have timeout at 10 mins then we know it failed because of slow restart of node.